lumicks/pylake/fitting/models.py

```diff
@@ -389,17 +391,17 @@ def ssdna(name, kilobases, kilobases_per_micron=0.56, temperature=24.53608821):
     ----------
     name : str
         Name for the model. This name will be prefixed to the model parameter names.
-    kilobases : integer
-        The number of kilobases in the sample
-    kilobases_per_micron: float
-        The number of kilobases evaluating to 1 micron. This is used to convert the kilobases to microns for the fit function.
+    dna_length_kb: integer
+        The length of the dna in the sample measured in kilobases
+    nm_per_b: float
+        The number of bases evaluating to 1 um. This is used to convert the length in kb to um as applied in the fit function.
     temperature: float
         The temperature in celsius. This is used to calculate the boltzmann * temperature value
     """
     import scipy.constants as spy
 
     model = freely_jointed_chain(name)
-    model.defaults[f"{name}/Lc"].value = kilobases * kilobases_per_micron
+    model.defaults[f"{name}/Lc"].value = dna_length_kb * nm_per_b
```

Comment:
Remember that `Lc` is in microns. I'd opt for `um_per_kb`.

Reply:
Ditto on the citation. Will update

Reply:
👍